vscode/snippet-generator/pom.xml

```diff
@@ -31,7 +31,11 @@
     <groupId>org.wso2.ballerina-integrator</groupId>
     <artifactId>ballerina-integrator-vscode-plugin</artifactId>
     <version>0.0.1-SNAPSHOT</version>
-    <packaging>jar</packaging>
+    <modules>
+        <module>snippet-generation-automation</module>
+        <module>auto-generated-files</module>
```

Comment:
isn't auto-generated-files module within the snippet-generation-automation module? Therefore do we need this module defined here?